.github/workflows/publish.yml

```diff
@@ -0,0 +1,31 @@
+name: Upload Python Package
```

Comment:
Is there a way to change and retrigger this for a release if it for some reason fails?

Reply:
Well if it fails, you can re run the build job?

Or you mean another thing?

Reply:
If I make a tag, publish the draft and this fails because we need to adjust the GitHub Actions file in some way. Would we have to remove the tag, fix the issue and re-add the tag again?

Reply:
Yeah something I considered as well hmmm

Yep, so we would have to remove the tag, make the fix, reupload the tag. 

I can add a RELEASE.md with the description, and leave there the commands for easy use.